src/parser/warrior/fury/modules/talents/Warpaint.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import { formatPercentage, formatThousands } from 'common/format';
+import TalentStatisticBox from 'interface/others/TalentStatisticBox';
+import Analyzer from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import { SELECTED_PLAYER } from 'parser/core/EventFilter';
+
+const REDUCTION_BONUS = 0.1;
+
+class Warpaint extends Analyzer {
+
+  damageMitigated = 0;
+  damageTaken = 0;
+
+  constructor(...args) {
+    super(...args);
+
+    this.active = this.selectedCombatant.hasTalent(SPELLS.WARPAINT_TALENT.id);
+
+    if(!this.active){
+      return;
+    }
+
+    this.addEventListener(Events.damage.to(SELECTED_PLAYER), this.onPlayerDamageTaken);
+  }
+
+  onPlayerDamageTaken(event) {
+    if (this.selectedCombatant.hasBuff(SPELLS.ENRAGE.id)) {
+      const preMitigatedDefensiveStance = (event.amount + event.absorbed) / (1 - REDUCTION_BONUS);
```

Comment:
is this copypasta from dstance or is there actually something going on with dstance here?

Reply:
Oops, forgot to rename that. Yes, I used the damage mitigation calculation from the dstance analyzer. I'll rename it now